### DISCUSSION TITLE
Update installed cask versions numbers after manual update

### DISCUSSION THREAD

**Discussion by mikkel1107** [Homebrew/discussions#3397] [Casks] [answered]
Is there a command to make brew update the current installed versions numbers of casks without auto-update? I often install applications with brew but I use another software to update moste but not all of my applications. This results in brew telling me that the updated applications is outdated even though it's the newest version which is installed.

**Comment by SMillerDev** [comment, marked as answer]
No, if you installed something through brew it'll keep track of the version you installed.
